legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/review/ReviewApi.java

```diff
@@ -53,12 +54,12 @@
      * Create a review for changes from the given workspace.
      *
      * @param projectId   project id
-     * @param workspaceId workspace id
+     * @param workspace   workspace
      * @param title       review title
      * @param description review description
      * @return new review
      */
-    Review createReview(String projectId, String workspaceId, String title, String description);
+    Review createReview(String projectId, Workspace workspace, String title, String description);
```

Comment:
We should not use Workspace here. We should take the workspace id as a String and some other piece of information to indicate the workspace type. I think we should add a WorkspaceType enum in the model.

Reply:
Fixed as above mentioned!